### PR TITLE
Small refactor

### PR DESCRIPTION
**Pull request by shikoku1**
- Change SendError to ValidationError 
- no need to put a trailing / on object aggregated send path

### CODE REVIEW

File: src/interfaces.rs [hunk 1]
```diff
@@ -168,34 +168,39 @@ impl Interfaces {
                     Interfaces::validate_float(obj.1)?;
 
                     let mapping = self
-                        .get_mapping(interface_name, &format!("{}{}", interface_path, obj.0))
-                        .ok_or_else(|| AstarteError::SendError("Mapping doesn't exist".into()))?;
+                        .get_mapping(
+                            interface_name,
+                            &format!("{}/{}", interface_path.trim_end_matches('/'), obj.0),
```
Comment (Annopaolo) [inline src/interfaces.rs:173]: Why trimming a `/` and re-introducing it in the format string?

Comment (shikoku1) [reply src/interfaces.rs:173]: so it works regardless if it was there or not

Comment (shikoku1) [reply src/interfaces.rs:173]: Can you tell me what this difference is

Comment (Annopaolo) [reply src/interfaces.rs:173]: Check this example (Elixir syntax):
```
iex(6)> Astarte.Device.send_datastream(pid, "org.astarte-platform.genericsensors.Geolocation", "/test/", %{"accuracy" => 42.42})
{:error, :invalid_path}
iex(7)> Astarte.Device.send_datastream(pid, "org.astarte-platform.genericsensors.Geolocation", "/test", %{"accuracy" => 42.42}) 
:ok
```

**Pull request closed without merging**